doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3505,10 +3505,11 @@ NCSDK-11432: DFU: Erasing secondary slot returns error response
 .. rst-class:: v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
 
 NCSDK-6238: Socket API calls might hang when using Download client
-  When using the :ref:`lib_download_client` library with HTTP (without TLS), the application might not process incoming fragments fast enough, which can starve the :ref:`nrfxlib:nrf_modem` buffers and make calls to the Modem library hang.
-  Samples and applications that are affected include those that use :ref:`lib_download_client` to download files through HTTP, or those that use :ref:`lib_fota_download` with modem updates enabled.
+  When using the :ref:`lib_download_client` and :ref:`lib_downloader` libraries with HTTP (without TLS), the application might not process incoming fragments fast enough, which can starve the :ref:`nrfxlib:nrf_modem` buffers and make calls to the Modem library hang.
```

Comment:
Does it make sense to mention the downloader library in known issues concerning older NCS tags?

Reply:
Not when the issue is fixed. Thanks, will remove.